src/custom/utils/operator/index.ts

```diff
@@ -231,8 +157,9 @@ async function _getJson(chainId: ChainId, url: string): Promise<any> {
     if (!response) {
       throw new Error(`Error getting query @ ${url}`)
     } else if (!response.ok) {
-      const errorResponse = await response.json()
-      throw errorResponse
+      // is backend error handled at this point
+      const errorResponse: ApiError = await response.json()
```

Comment:
@W3stside important, can we assume the API will return always an error with the shape we expect?
If it's the case I don't say anything, but probably some cases where `!response.ok` won't return the details? 

For example, what happens if they return a `429`? 
Will we have that ApiError data you expect here?
![image](https://user-images.githubusercontent.com/2352112/118637555-3a10dc80-b7d6-11eb-90ad-f4998a0f8026.png)

Maybe you should use duck typing to know if it's what you expect?